countries/universal.js

```diff
@@ -1377,7 +1377,7 @@ const universalCountries = [
     name: "Antarctic Flyover",
     code: "Antarctic Flyover",
     country_code: "ANF",
-    two_letter_country_code: "AU",
+    two_letter_country_code: undefined,
```

Comment:
this does not seem to be a country. why is it here?

Reply:
It's kind of a region instead of a country, provided by nib I guess

Reply:
Let's do a quick call about this. It may come back to bite us. There's other cases where we'll run into issues with mapping a 2 letter country code will trip us up in the same way - eg. `ID` maps back to both Bali and Indonesia.

` 
{
    name: "Indonesia",
    code: "Indonesia",
    country_code: "IDN",
    two_letter_country_code: "ID",
},
...
{
    name: "Bali",
    code: "Bali",
    country_code: "BAL",
    two_letter_country_code: "ID",
}
,

Another solution could be to have an attribute like `common_mistake: true` for the cases that don't actually match a country. Or `exact_match: true|false` for all cases. Would need to understand where these are used and where they came from first though. A call would be easiest.